### ISSUE TITLE
Question about class thresholds

### ISSUE THREAD

**Issue by vishaal27** [xinyu1205/recognize-anything#137]
I am trying to run RAM++ on open-set classes. I noticed that you have two files: `ram_tag_list_threshold.txt` and `ram_tag_list.txt`. I assume these are confidence thresholds for each of the classes in the tag list file. How did you come up with these thresholds? How can I adapt them for my own open-set setting?

**Comment by xinyu1205** [comment]
Thanks for your attention.
For seen categories, we set the default threshold as 0.68, and tune the thresholds for various categories based on as many test datasets as possible.
Our experiments indicate that the logits for open-set categories typically lower than the seen categories (see RAM++ paper Figure 6). Therefore, the thresholds for open-set categories should set lower than the seen categories.
You can fine-tune the thresholds based on some visualization examples or your own test datasets.
Best wishes.

**Comment by vishaal27** [comment]
Awesome, thanks, Fig. 6 is really helpful!
